docs/adr/0001-update-logins-api.md

```diff
@@ -0,0 +1,96 @@
+# Update Logins API
+
+* Status: proposed
+* Date: 2021-05-27
+
+Technical Story: [#4101](https://github.com/mozilla/application-services/issues/4101)
+
+## Context and Problem Statement
+
+The per-field encryption in autofill, which we are planning to replicate in logins, separates the storage and encryption logic by limiting the storage layer to the management of encrypted data.
+
+Applying this approach in logins will break the existing validation and deduping code so we need a way to implement per-field encryption while supporting the validation and de-duping behavior.
+
+
+## Decision Drivers
+
+* Avoiding the implementation of new security approaches that may require additional time and security resources
+* Establishing a standard encyrption approach across components
+
+
+## Considered Options
+
+* Option 1 - Reduce the API functions that require the encryption key and pass the key to the remaining functions
+* Option 2 - Implement a different key management approach
```

Comment:
I think "Implement a different" is misleading - it's actually more like "Implement the same"! I'd reword this as something like "Keep the general shape of the API that's in place now - the app can pass the encryption key at any time to "unlock" the API, and re-lock it at any time, but the API in its entirety is only available when unlocked."